index.html

```diff
@@ -55,9 +56,21 @@
     <!-- Main jumbotron for a primary marketing message or call to action -->
     <div class="jumbotron">
       <div class="container">
-        <h1>Hello, world!</h1>
+
+          <div class="register-container">
+            <div class="new-account">Register Account</div>
+              <input type="first-name" placeholder="First Name" class="form-control">
+              <input type="last-name" placeholder="Last Name" class="form-control">
+              <input type="email" placeholder="Email" class="form-control">
+              <input type="password" placeholder="Password" class="form-control">
+              <button type="submit" class="btn btn-submit">Submit</button>
+          </div>
+
```

Comment:
Should we add some form validation to this? We want to ensure that valid emails are being entered and that passwords are meeting security requirements.